modules.d/95nvmf/module-setup.sh

```diff
@@ -25,17 +25,27 @@ check() {
         [[ $trtype == "fc" ]] || [[ $trtype == "tcp" ]] || [[ $trtype == "rdma" ]]
     }
 
+    has_nbft() {
+        local f found=
+        for f in /sys/firmware/acpi/tables/NBFT*; do
+            [ -f "$f" ] || continue
+            found=1
+            break
+        done
+        [[ $found ]]
+    }
+
     [[ $hostonly ]] || [[ $mount_needs ]] && {
         pushd . > /dev/null
```

Comment:
I know this is not part of the PR, but does anyone know why `pushd .` is here? I think this is basically a no-op.

Reply:
Is it just checking that `for_each_host_dev_and_slaves` does not run `popd`? I'd love to know the reasoning behind this implementation.

Reply:
It is indeed not part of this PR, therefore I'd like to close this conversation. 

AFAICS, the `pushd/popd` code (like lots of the initial nvmf code) has been copied from `95iscsi/module-setup.sh`, where it had been added by @haraldh in commit b093aa2 ("beautified shell code") 10y ago, without further explanation. Perhaps at that time `for_each_host_dev_and_slaves` could change the working directory? I don't know.

Reply:
Right, good to know the origin of it. Thanks for the response!